### DISCUSSION TITLE
Security fore phone

### DISCUSSION THREAD

**Discussion by Ryjden** [community/community#134569] [New to GitHub]
How to see if my phone wachet another person?

**Comment by ghostinhershell** [comment]
Hi @Ryjden, we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by ibrartalab** [comment]
@Ryjden 
Hi, the simple answer to your question is no one can hack or even watch your phone in case they have your active email access , if you think this is the situation then immediately change your password and log put from all those devices that not in your use.
